### DISCUSSION TITLE
Version 8.2.0.23 - copy URL no longer works?

### DISCUSSION THREAD

**Discussion by aarrgh123** [aclap-dev/video-downloadhelper#131] [Help and Q&A]
### Your question

Edit: sorry it was explained in anothere topic

**Comment by AxSlingerland** [comment]
In the meantime while they're fixing **Copy URL**, you can copy the URL by clicking on **Details** instead (above **Copy URL**), and then copying the **mediaManifest** field. I know that is more clicks and takes a few seconds longer but you end up with the same URL.

**Comment by paulrouget** [comment]
Hi! We are aware of that issue, and it will be fixed in the upcoming version (this weekend).

Sorry for the bug :|

**Reply by AxSlingerland** [reply]
I knew that... I knew you would fix it and I knew you also have lives to lead. Have a great weekend!

**Reply by Mattmjs** [reply]
If VDH can already perform the download, why does everyone want copy url. Is it used in other tools / software ?

**Comment by AxSlingerland** [comment]
Well, personally, I don't use the co-app because it caused some problems with my hard drive. (Files and folders that can't be deleted, etc.) So I used Copy URL instead. It was basically the same amount of effort as using the co-app download feature. Doing what I suggested instead of using the Copy URL (which is down at the moment, but they're working on it) is just a few extra clicks to do the same thing. I then paste the URL into a batch file and download it with yt-dlp.

**Reply by GrampaWildWilly** [reply]
Failure to delete objects is a rare occurrence.  Yes it happens, but hardly ever.  I think perhaps you were being a bit drastic to condemn the CoApp on that basis.  The cure for inability to delete things is usually a `chkdsk /f`.  But to each his own, I suppose.

**Reply by AxSlingerland** [reply]
I tried it again with different files and different folders and had the same thing happen again. However, I just said I don't use it, I did not condemn it. I don't particularly like Ford's, but that doesn't mean they aren't good vehicles, just that I don't particularly like them. The failure to delete those files and folders was probably just a glitch that could easily be fixed, or more likely it was this POS laptop I spent too much money on, but I prefer yt-dlp anyway so it was no big deal. I'm a Grampa too, and have nearly 30 years experience using computers, so I know about chkdsk too. Although, I'm not too wild anymore! :)

**Comment by skaox** [comment]
> In the meantime while they're fixing **Copy URL**, you can copy the URL by clicking on **Details** instead (above **Copy URL**), and then copying the **mediaManifest** field. I know that is more clicks and takes a few seconds longer but you end up with the same URL.

Hi,
yes sure can do that but if you have a lot of tabs like me (-: I know it's my fault ;-)
this one (details/info) will not be open just after the one you are it will be at the end (at right).
Same thing when you close this tab it will not go back to the tab where you click on Details.
It's not the end of the world just loosing a bit of time.

**Came here to report the same problem with Copy URL**
Ubuntu Mate 20.04/22.04  -  Firefox 122.0.1  -  Video DownloadHelper 8.2.0.23

Updated recently vdhcoapp-linux-x86_64.deb  (1.x -> 2.0.17).
Nothing to report about not possible to move or delete files on my linux. 
I don't use it, prefer to copy url with vdh then uGet will detect it easy peasy.

By the way a big THANK YOU for what you're doing with a giant Big Up !

**Comment by AxSlingerland** [comment]
To be honest, I'm not really sure what you're talking about... But if you mean doing many downloads in a batch, I do that too. It does take a bit of time to create a list, but once you start it, you can usually forget about it and just let it do it's thing. Sometimes there are files on a slow server that you have to babysit, or you may even have to download from a different source, but those are isolated incidents.

**Comment by skaox** [comment]
(-: Not sure we're talking about the same thing ...
Me it's **not** [wget](https://www.gnu.org/software/wget/) and you ?
No batch downloads either.

[uGet](https://ugetdm.com/) is a download manager for lin/mac/win
free & open source  https://github.com/ugetdm
paid equivalent are idm for windows or folx for mac.

uget will detect the url past into clipboard and will propose to download it.
Possible to set a number of connections, active downloads, rename, use a list, etc .... 

uget & vdh or web browser work together one for having the link/url of a mkv/mp4 video file and
the other for downloading there's also a last one named vlc for watching it before download is finish.

This is why I said uGet will detect it easy peasy ;-)
